code/modules/atmospherics/machinery/other/meter.dm

```diff
@@ -58,6 +58,9 @@
 	if(!target)
 		icon_state = "meterX"
 		return 0
+	else if(!(target.flags_1 & INITIALIZED_1))

```

Comment:
This does exactly the same thing as section above, change the condition instead.